backend/fms/views.py

```diff
@@ -1,5 +1,7 @@
+from django.http import JsonResponse
```

Comment:
not sure if JsonResponse is necessary, but since it is a JsonResponse, might as well keep it :)